### ISSUE TITLE
"Use explicit type instead of 'var'" doesn't fire for deconstructed tuples

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#23752]
``` C#
    class Program
    {
        static void Main(string[] args)
        {
            var (foo, bar) = GetFooBar();
        }

        private static (DateTime foo, DateTime bar) GetFooBar()
        {
            return default((DateTime foo, DateTime Bar));
        }
    }
```

![image](https://user-images.githubusercontent.com/1103906/33919165-42494082-e00b-11e7-98dd-9d328e387152.png)

**Expected:** 
```
Message	IDE0008	Use explicit type instead of 'var' 
```
Which expands to:
``` C#
(DateTime foo, DateTime bar) = GetFooBar();
```
**Actual:** No message

Note: It does fire for:

``` C#
var fooBar = GetFooBar();
```

**Comment by dpoeschl** [comment]
Should this convert it to `(DateTime foo, DateTime bar) = GetFooBar();` to make it "explicit"?

**Comment by davkean** [comment]
Yes that's correct.

**Comment by jcouv** [comment]
Fix is merged for 15.6 preview 4
